tools/scripts/upgrades/start-node.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash  
+
+# The script starts node locally (no LocalNet) with cosmovisor and waits for an upgrade.
+# Cosmovisor: https://docs.cosmos.network/main/build/tooling/cosmovisor
+# Install with `go install cosmossdk.io/tools/cosmovisor/cmd/cosmovisor@v1.5.0`
+
+export DAEMON_NAME=poktrolld 
+# Pre-build binaries. Built with `ignite chain build --skip-proto --output .`
```

Comment:
#PUC It's unclear what the expectations are from the user on prior configuration, etc...

Leaving one comment but I think you get the gist.

Reply:
Added more context in comments.